### PR TITLE
Conan Tutorial Docs

### PR DESCRIPTION
**Pull request by gazzatav**
This pull request mostly corrects typos and adds missing words in the sections about consuming and creating packages.  It also restructures some paragraphs where I, as a native English speaker could see two or three possible meanings for the original or found the original difficult to understand.  If I have still misunderstood perhaps this can serve to show how and where the original could be improved.

### CODE REVIEW

File: tutorial/creating_packages/build_packages.rst [hunk 1]
```diff
@@ -87,8 +87,8 @@ Changes introduced in the recipe
   - We use the ``tools.build:skip_test`` configuration in the ``build()`` method,
     after building the package and tests, to decide if we want to run the tests or not.
   
-  - In this case we are using **gtest** for testing and we have to add the check if the
-    build method to run the tests or not, but this configuration also affects the
+  - In this case we are using **gtest** for testing and we have to check if the
+    build method is to run the tests or not  This configuration also affects the
```
Comment (memsharded) [inline tutorial/creating_packages/build_packages.rst:91]:
```suggestion
    build method is to run the tests or not.  This configuration also affects the
```

**Review by memsharded — APPROVED** [review]
Looks good, thanks very much for taking the time to improve the docs!

File: tutorial/creating_packages/build_packages.rst [hunk 2]
```diff
@@ -87,8 +87,8 @@ Changes introduced in the recipe
   - We use the ``tools.build:skip_test`` configuration in the ``build()`` method,
     after building the package and tests, to decide if we want to run the tests or not.
   
-  - In this case we are using **gtest** for testing and we have to add the check if the
-    build method to run the tests or not, but this configuration also affects the
+  - In this case we are using **gtest** for testing and we have to check if the
+    build method is to run the tests or not.  This configuration also affects the
```
Comment (RubenRBS) [inline tutorial/creating_packages/build_packages.rst:91]:
```suggestion
    build method is to run the tests or not. This configuration also affects the
```

**Review by RubenRBS — APPROVED** [review]
Thanks a lot!! We really appreciate it :)

**Review by czoido — APPROVED** [review]

**Pull request merged**